packages/devtools_app/lib/src/shared/console/widgets/description.dart

```diff
@@ -382,14 +382,16 @@ class DiagnosticsNodeDescription extends StatelessWidget {
         }
       }
 
-      if (!diagnosticLocal.isSummaryTree &&
-          diagnosticLocal.isCreatedByLocalProject) {
-        textStyle = textStyle.merge(DiagnosticsTextStyles.regularBold);
```

Comment:
should we still be bolding the nodes that are from the local project or do you think using the subtle text style for the others is enough distinction?

Reply:
Ah you're right this shouldn't have been removed! This was bolding the local widgets in the **details** tree (not the summary tree). I've added this back in with a TODO to remove it once the details tree is gone.